pkg/stomp/StompContext.php

```diff
@@ -38,10 +38,16 @@ class StompContext implements Context
      */
     private $stompFactory;
 
+    /**
+     * @var bool
+     */
+    private $transient;
+
     /**
      * @param BufferedStompClient|callable $stomp
+     * @param mixed                        $transient
      */
-    public function __construct($stomp, string $extensionType)
+    public function __construct($stomp, string $extensionType, $transient = true)
```

Comment:
Connections start out as "transient" which means that they will use disconnect behaviour.  I have it here in the constructor as configurable if for whatever reason a connection needs to start out long-lived.